From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 14 Mar 2024 13:41:51 -0500
Subject: [PATCH] Force opening remote files with kio-fuse

Due to a known bug opening media files over a remote share fails to load them correctly over KIO. Edit the desktop file so that Plasma thinks Haruna only supports local files which will cause it to use kio-fuse to load remote files.

https://bugs.kde.org/show_bug.cgi?id=469457
---
 data/org.kde.haruna.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.kde.haruna.desktop b/data/org.kde.haruna.desktop
index 8c5ff43..717115a 100644
--- a/data/org.kde.haruna.desktop
+++ b/data/org.kde.haruna.desktop
@@ -37,7 +37,7 @@ Name[uk]=Мультимедійний програвач Haruna
 Name[x-test]=xxHaruna Media Playerxx
 Name[zh_CN]=Haruna 媒体播放器
 Name[zh_TW]=Haruna 媒體播放器
-Exec=haruna %u
+Exec=haruna %f
 Icon=haruna
 Type=Application
 Categories=Qt;KDE;AudioVideo;Player;Video;TV;
